.github/workflows/generate-reports.yml

```
@@ -0,0 +1,61 @@
+# This workflow will generate surefire reports and API documentation with Maven and publish
+# them on the 'gh-pages' branch of the target Github repository.
+
+name: Generate reports and API documentation
+
+on:
+  release:
+    types: [created]
+  push:
+    branches:
+      - development
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install git
+        run: sudo apt-get install git
+
+      - name: Set up JDK 1.11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.11
```

I think we should keep the JDK version consistent for all github actions 
```suggestion
          java-version: 1.8
```